frontend/src/lib/components/VersionChecker/versionCheckerLogic.ts

```diff
@@ -98,7 +98,11 @@ export const versionCheckerLogic = kea<versionCheckerLogicType>([
                         ORDER BY latest_timestamp DESC
                         limit 10`
 
-                    const res = await api.SHAMEFULLY_UNTAGGED_queryHogQL(query, { refresh: 'force_blocking' })
+                    const res = await api.queryHogQL(
+                        query,
+                        { scene: 'Settings', productKey: 'platform_and_support' },
```

Comment:
I think the Scene here is not quite right, this can show up on many different scenes (I think it shows up in the web analytics Scene, as an example). Can you grab the parent scene from Kea somehow?